app/components/post_list/post_list.js

```diff
@@ -260,6 +260,7 @@ export default class PostList extends PureComponent {
         // Remember that the list is rendered with item 0 at the bottom so the "previous" post
         // comes after this one in the list
         const previousPostId = index < this.props.postIds.length - 1 ? this.props.postIds[index + 1] : null;
+        const beforePrevPostId = index < this.props.postIds.length - 1 ? this.props.postIds[index + 2] : null;
```

Comment:
Possibly doesn't matter too much since `undefined` would currently be the result, but if you want the value of `beforePrevPostId` to specifically be `null` when `index + 2` is out of range, you may want to do the following to balance the fact that you're looking 2 indexes ahead of the current index in the array.

```suggestion
        const beforePrevPostId = index < this.props.postIds.length - 2 ? this.props.postIds[index + 2] : null;
```

Reply:
Thanks I applied these changes! Good catch!